.gitignore

```diff
@@ -1,4 +1,6 @@
-vendor
-auth
-server.crt
-server.key
+/.idea/
```

Comment:
What's the content of .idea directory?
Don't add manifests to gitignore. The idea is to have a basic set of manifests in the repo.

Reply:
/.idea is for intellij. Having it excluded makes it easier for collaborators like me to contribute to your code base.

Manifests are a build artifact. Build artifacts should never be committted because they depend on the source code, in this case the configuration. My configuration is different from yours, so it results in different manifests and without excluding them git wants to commit them, so they would be in this pull request. That would be bad because I really only changed a few lines of code.

Reply:
I'd like to keep the manifests since they provide a good base for lots of use-cases and no compilation and tools install are required in this case.
I might add a few info to the readme about this.

Reply:
Well, this is your repo, but I don't get your point: Your README says to do a `make deploy`, which (re)generates the manifests anyway. Plus, this repo cannot be used without customization, so you really have to call `make deploy`. Therefore, I conclude the manifests are compilation artifacts (just like Java byte code) and hence should not be committed.

I wouldn't bother if the manifests were the same for all users, but they depend on the individual configuration, so anyone who wants to collaborate with you will run into the same issue like me.

Reply:
Check https://github.com/carlosedp/cluster-monitoring/commit/0d032eac7f083471e5ff0f081188748d5635854f. It makes easier for the user to just change the Ingress URLs without a full rebuild thus not requiring install of all Golang tools.

Reply:
Well, this idea falls short for anyone who's master node's IP is not exactly `192.168.15.15` or who doesn't want `192.168.15.15.nip.io` in their ingress URLs or use any of the plugin modules you are kindly offering.

Anyway, you can close my PR if you don't want these changes.

Reply:
It's actually two different issues we are discussing. The origin of the PR to fix the TLS problem is very welcome.
Removing the default manifests from the repo is not required since I addressed a way to change the Ingress URLs with a Makefile target avoiding a rebuild.